codegen/java-gen/src/main/scala/com/lightbend/akkasls/codegen/java/EntityServiceSourceGenerator.scala

```diff
@@ -197,20 +367,16 @@ object EntityServiceSourceGenerator {
       ) <> line <>
       line <>
       "/** An event sourced entity. */" <> line <>
-      "@EventSourcedEntity" <> parens(
-        "entityType" <+> equal <+> dquotes(entityType)
-      )
-      <> line <>
       `class`("public", s"$className extends $interfaceClassName") {
         "@SuppressWarnings" <> parens(dquotes("unused")) <> line <>
         "private" <+> "final" <+> "String" <+> "entityId" <> semi <> line <>
         line <>
         constructor(
           "public",
           className,
-          List("@EntityId" <+> "String" <+> "entityId")
+          List("EventSourcedContext" <+> "context")
```

Comment:
Value entity is only passing the `String entityId`. We need to align this.

The context is not so useful right now. It only has the entityId and the `serviceCallFactory`, but the `serviceCallFactory` is already available from the `CommandContext`. 

We probably want to have something better than the exiting (untyped) `ServiceCallFactory` and then we want it to be injected in the constructor so users can instantiate clients at construction time (and not per command using the CommandContext). 

So, passing the context is probably something we may need to keep (and introduce in VE), but not yet clear how we will really use it. 

On the other hand, passing context make unit testing harder. It will be better to not have a mock context of any kind.

Reply:
ok, there are more things that are different, I'll align them